spec/openai/client/http_spec.rb

```diff
@@ -286,4 +286,27 @@
       }
     end
   end
+
+  describe 'extra_headers' do
+    describe '#chat' do
+      let(:client) { OpenAI::Client.new }
+      let(:extra_headers) { { 'Helicone-Property-Conversation': 'conversation-123' } }
+      let(:chat_params) do
+        {
+          parameters: {
+            model: 'gpt-3.5-turbo',
+            messages: [],
+            temperature: 0.1,
+            extra_headers: extra_headers
+          }
+        }
+      end
+
+      it 'includes extra headers in the chat request' do
+        expect(client).to receive(:chat).with(chat_params)
+
+        client.chat(chat_params)
```

Comment:
This test doesn't test anything really - it expects line 308 to happen, and then line 308 happens, causing it to pass.

Better to test a bit deeper, ie check that the headers actually get set. Can't see an easy way to do that right now